src/Data/PQueue/Internals.hs

```diff
@@ -70,7 +70,7 @@ instance (Ord a, Data a) => Data (MinQueue a) where
   dataTypeOf _ = queueDataType
 
 queueDataType :: DataType
-queueDataType = mkDataType "Data.PQueue.Min.MinQueue" [emptyConstr, consConstr]
+queueDataType = mkDataType "Data.PQueue.Internals.MinQueue" [emptyConstr, consConstr]
```

Comment:
I'm not sure about this. Of course it is actually defined in `Data.PQueue.Internals`, but logically it's defined in `Data.PQueue.Min`. Users shouldn't need to care about the internal modules imo.

Reply:
I got nervous about [`liftData`](https://hackage.haskell.org/package/template-haskell-2.18.0.0/docs/Language-Haskell-TH-Syntax.html#v:liftData), but fortunately that gets its module (and package) names from the `TypeRep`, not from any `Data` junk. So it's probably fine to leave that as `Data.PQueue.Min`; I'll make the change.